<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntegrationInputLotForm</class>
 <widget class="GUIFormBase" name="IntegrationInputLotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditLotNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSubCode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonGenerateLot">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLotNumber">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubCode">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLotID">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLotID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTitleLotID">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTitleLotNumber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTitleSubCode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sub code</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditLotID</tabstop>
  <tabstop>pushButtonLotID</tabstop>
  <tabstop>lineEditLotNumber</tabstop>
  <tabstop>pushButtonLotNumber</tabstop>
  <tabstop>lineEditSubCode</tabstop>
  <tabstop>pushButtonSubCode</tabstop>
  <tabstop>toolButtonGenerateLot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
